src/TestData.hs

```diff
@@ -29,6 +39,49 @@ data Account ts = Account
 deriving instance AllHave Show (Account ts) => Show (Account ts)
 deriving instance AllHave Eq (Account ts)   => Eq (Account ts)
 
+jonathan :: User 'Data
```

Comment:
Can we move this stuff into a proper test?

Reply:
i'd still like to move this out of Testdata. in my ideal world there will be no testdata file!